apps/rush-lib/src/cli/test/basicAndRunBuildActionRepo/b/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "b",
   "version": "1.0.0",
   "description": "Test package b",
+  "dependencies": {
```

Comment:
Unrelated?

Reply:
It's a fix for a test failure that turned out to be a race condition.  Making `b` dependent on `a` forces the build to happen in a specific order.